### PR TITLE
Apply conventions to cucumber (core)

### PR DESCRIPTION
**Pull request by Bischoff**
Split step "execute some tests" into several precise steps
Consistent conventions for features, scenarios, and steps
Correct English
Resolved duplicate features
Removed unused steps

### CODE REVIEW

**Comment by MalloZup** [comment]
@Bischoff thx, i will review it :rose:

File: features/core_min_bootstrap.feature [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
+     When I go to the minion onboarding page
      Then I should see a "accepted" text
      And the salt-master can reach "sle-minion"
-     And I navigate to "rhn/systems/Overview.do" page
+     When I navigate to "rhn/systems/Overview.do" page
      And I wait until I see the name of "sle-minion", refreshing the page
      And I wait until onboarding is completed for "sle-minion"
 
-  Scenario: Run a remote command sles-minion (salt-service)
+  Scenario: Run a remote command on normal SLES minion
```
Comment (MalloZup) [inline features/core_min_bootstrap.feature]: is normal not too neutral?

File: features/core_min_bootstrap.feature [hunk 2]
```diff
@@ -66,44 +66,44 @@ Feature: register a salt-minion via bootstrap
...
      And I select "1-MINION-TEST" from "activationKeys"
      And I click on "Bootstrap"
-     Then I wait until i see "Successfully bootstrapped host! " text
+     Then I wait until I see "Successfully bootstrapped host! " text
      And I navigate to "rhn/systems/Overview.do" page
      And I wait until I see the name of "sle-minion", refreshing the page
      And I wait until onboarding is completed for "sle-minion"
 
-  Scenario: verify minion bootstrapped with activation key, packages
+  Scenario: Verify that minion bootstrapped with Salt key and packages
```
Comment (MalloZup) [inline features/core_min_bootstrap.feature]: good catch then :)

File: features/step_definitions/docker_steps.rb [hunk 3]
```diff
@@ -90,7 +90,7 @@ def retrieve_minion_id
   visit("https://#{$server.full_hostname}/rhn/manager/cm/build")
 end
 
-And(/^I verify that all "([^"]*)" container images were built correctly in the gui$/) do |count|
+And(/^all "([^"]*)" container images should be built correctly in the GUI$/) do |count|
```
Comment (MalloZup) [inline features/step_definitions/docker_steps.rb]: Here should be All --> big A

File: features/step_definitions/xmlrpc_common.rb [hunk 4]
```diff
@@ -259,18 +259,6 @@
   detail = virtualhostmanager.getDetail(label)
 end
 
-Then(/^I should get two modules$/) do
```
Comment (MalloZup) [inline features/step_definitions/xmlrpc_common.rb]: are this not used? then good to remove them! good catch

File: features/core_min_bootstrap.feature (hunk 1 above)
Comment (Bischoff) [reply features/core_min_bootstrap.feature]: that's the way we speak in the office... normal = Salt service (not SSH nor reverse tunnel)

File: features/core_min_bootstrap.feature (hunk 2 above)
Comment (Bischoff) [reply features/core_min_bootstrap.feature]: Yeah, from steps below it was not an activation key but a Salt key

File: features/step_definitions/docker_steps.rb (hunk 3 above)
Comment (Bischoff) [reply features/step_definitions/docker_steps.rb]: Why not. English would command lower case after "then" or "and", but we can make it a convention

File: features/step_definitions/xmlrpc_common.rb (hunk 4 above)
Comment (Bischoff) [reply features/step_definitions/xmlrpc_common.rb]: Please double-check that...

Comment (MalloZup) [reply features/step_definitions/xmlrpc_common.rb]: ok they are not used +1

File: features/step_definitions/docker_steps.rb (hunk 3 above)
Comment (Bischoff) [reply features/step_definitions/docker_steps.rb]: I tried, it looks ugly with upper case

**Pull request merged**